bower.json

```diff
@@ -28,13 +28,13 @@
   ],
   "dependencies": {
     "polymer": "^2.0.0",
-    "vaadin-themable-mixin": "vaadin/vaadin-themable-mixin#^1.4.1",
-    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.0.0",
-    "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.3.0",
-    "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.2.0",
+    "vaadin-themable-mixin": "vaadin/vaadin-themable-mixin#^1.5.2",
+    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.3.1",
+    "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.6.0",
+    "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.3.2",
     "vaadin-text-field": "vaadin/vaadin-text-field#^2.3.0",
     "vaadin-button": "vaadin/vaadin-button#^2.1.0",
```

Comment:
There is an update to `button` with rtl styles. I guess we should make a release and update this dependency, just not to loose it. 😅 
https://github.com/vaadin/vaadin-button/commit/1098bbf7ca55bb76286ec5418b1b5796907f92fb

Reply:
Updated.